pkg/nodebase/conf/allowlist.js

```diff
@@ -0,0 +1,19 @@
+export * from '../../core/utils.min.js';
+export * from '../../Library/App/Worker/App.js';
```

Comment:
I don't think App goes here. Who is importing it?

Reply:
i copied it from exploder app, removed now

Reply:
should i remove it from `exploder` as well, right?